### PR TITLE
Add simple login screen

### PR DESCRIPTION
**Pull request by MRZA-MRZA**
This adds simple login screen where you can enter your JID and password.

### CODE REVIEW

**Comment by LNJ2** [comment]
@MRZA-MRZA Please keep in mind that everything has to be rewritten in Kirigami.

File: app/resources/LoginPage.qml [hunk 1]
```diff
@@ -0,0 +1,48 @@
+import QtQuick 2.5
+import QtQuick.Controls 1.4
```
Comment (geobra) [inline app/resources/LoginPage.qml]: On Debian 8 stable, only QtQuick.Controls 1.2 is available

Comment (MRZA-MRZA) [inline app/resources/LoginPage.qml]: Changed version. I'll try to use Kirigami next time.

**Comment by JBBgameich** [comment]
As far as we haven't succesfully built kirigami for sailfish OS, we probably should do as much as we can only with QtQuick.Controls. I know the app will not look really nice then, but this should be made when we have decided on the UI framework.

File: app/source/Kaidan.cpp [hunk 2]
```diff
@@ -10,9 +10,29 @@
...
 {
-	client = new Swift::Client("jid@...", "pass", networkFactories);
+	netFactories = networkFactories;
+	connected = false;
+}
+
+Kaidan::~Kaidan()
+{
+	if(connected){
+		client->removePayloadSerializer(&echoPayloadSerializer);
```
Comment (LNJ2) [inline app/source/Kaidan.cpp]: In C++ I would like this style more:
```
if (connected)
{
        client->...
}
```

**Comment by LNJ2** [comment]
FYI: I built it: no errors.

File: app/resources/qml.qrc [hunk 3]
```diff
@@ -1,5 +1,7 @@
 <RCC>
-	<qresource prefix="/">
-		<file>main.qml</file>
-	</qresource>
+    <qresource prefix="/">
+        <file>main.qml</file>
+        <file>RosterPage.qml</file>
+        <file>LoginPage.qml</file>
+    </qresource>
```
Comment (MRZA-MRZA) [inline app/resources/qml.qrc]: I think Qt Creator ignores indentation in .qrc files. I can fix it manually if you want.

File: app/resources/LoginPage.qml [hunk 4]
```diff
@@ -0,0 +1,48 @@
+import QtQuick 2.5
+import QtQuick.Controls 1.2
+import harbour.kaidan 1.0
+
+Item{
+	Column{
+		Row{
+			TextField{
```
Comment (MRZA-MRZA) [inline app/resources/LoginPage.qml]: https://github.com/KaidanIM/Kaidan/issues/23
Please, share your opinion.

**Comment by MRZA-MRZA** [comment]
@geobra Please check what errors you get in Application output.

File: app/resources/qml.qrc (hunk 3 above)
Comment (LNJ2) [inline app/resources/qml.qrc]: YES, I want.

**Comment by geobra** [comment]
It works 'out of the box' within debian testing. On debian stable, no controls will be displayed. So no possbility to enter jid and passwort to log in. But as this is only a temporary UI before we switch to kirigami, it's okay for me.

**Pull request merged**